app/account/forms.py

```diff
@@ -112,3 +114,32 @@ class ChangeEmailForm(Form):
     def validate_email(self, field):
         if User.query.filter_by(email=field.data).first():
             raise ValidationError('Email already registered.')
+
+
+class EditProfileForm(Form):
+    first_name = StringField('First name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    last_name = StringField('Last name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    affiliation_options = [
+        ('1', 'Veteran'),
+        ('2', 'Active Duty'),
+        ('3', 'National Guard'),
+        ('4', 'Reservist'),
+        ('5', 'Spouse'),
+        ('6', 'Dependent'),
+        ('7', 'Family Member'),
+        ('7', 'Supporter'),
+        ('8', 'Other')]
```

Comment:
Let's remove this affiliation array thingy. Then turn the SelectMultipleField into a `QuerySelectMultipleField`. I'm like 70% sure that exists.